stock_picking_delivery_info_computation/README.rst

```diff
@@ -0,0 +1,115 @@
+=======================================
+Stock Picking Delivery Info Computation
+=======================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fdelivery--carrier-lightgray.png?logo=github
+    :target: https://github.com/OCA/delivery-carrier/tree/12.0/stock_picking_delivery_info_computation
+    :alt: OCA/delivery-carrier
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/delivery-carrier-12-0/delivery-carrier-12-0-stock_picking_delivery_info_computation
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/99/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module extends the functionality of delivery to support
+calculating the picking weight giving priority to the package operations and
+also allowing you to calculate volume manually with same logic.
+
+On a picking which only has inital demand, calculation will be as standard,
+based on the picking lines product quantity.
+
+As soon as you have any reserved quantity, it will only take them into account.
+In this case, the computation of the weight and volume is based in *to do*
+quantities, unles you introduce anything in *done* quantity, switching to this
+column in that case.
+
+The volume is auto-computed when the picking is generated from a sales order,
+or a backorder is created from original picking, but remains editable while
+the picking is not validated for any possible manual edition.
+
+You also have a button available while the picking is not validated for
+recomputing volume with current data.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Inventory > All Transfers* and pick one.
```

Comment:
All transfers* is not valid.  Only draft, waiting and ready. If the transfer is Done or cancelled the button doesn't appears

Reply:
This is because README must be adapted to v12